sample_app/main_linux.c

```diff
@@ -857,6 +828,118 @@ int write_bool_to_file(const char*  filepath, bool value)
    return 0;
 }
 
+/**
+ * Print contents of a buffer
+ *
+ * @param bytes      In: inputbuffer
+ * @param len        In: Number of bytes to print
+*/
+void print_bytes(uint8_t *bytes, int32_t len)
+{
+   printf("  Bytes: ");
+   for (int i = 0; i < len; i++)
+   {
+      printf("%02X ", bytes[i]);
+   }
+   printf("\n");
+}
+
+/**
+ * Copy an IP address (as an integer) to a struct
+ *
+ * @param destination_struct  Out: destination
+ * @param ip                  In: IP address
+*/
+void copy_ip_to_struct(pnet_cfg_ip_addr_t* destination_struct, uint32_t ip)
+{
+   destination_struct->a = (ip&0xFF);
```

Comment:
nitpicking but please use space between operator and arguments, i.e. (ip & 0xFF)